### PR TITLE
Added-missing-examples-of-Servo-library

### LINKED ISSUE #51: Absence of examples of some methods
While browsing through the codebase, I came to know that examples of some functionality in the Servo library are not available. Below are the methods for which examples are needed.
-> servo.writeMicroseconds()
-> servo.read()
-> servo.attached()
-> servo.detach()

### PR DESCRIPTION
**Pull request by Engineer1999**
This pull request is in answer to the issue #51 . I have added 2 examples which includes below-mentioned methods. 
-> servo.writeMicroseconds()
-> servo.read()
-> servo.attached()
-> servo.detach()
Examples have been tested on tinkercad and work as expected.

### CODE REVIEW

File: examples/MultipleServoMotors/MultipleServoMotors.ino [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  Serial.println("Servo motor attached to pin 12");
+  
+  myservo.write(0);
+  delay(1000);
+  myservo.write(90);
+  delay(1000);  
+  myservo.write(180);
+  delay(1000);
+  } else{
+    Serial.println("Servo motor attachement failed");
```
Comment (per1234) [inline examples/MultipleServoMotors/MultipleServoMotors.ino]:
```suggestion
    Serial.println("Servo motor attachment failed");
```

File: examples/MultipleServoMotors/MultipleServoMotors.ino [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+  
+  myservo.detach();        // detach the servo motor
+  if (myservo.attached()){ // cheack whether servo motor is attached or not
+    Serial.println("Servo motor is still attached");
+  } else{
+    Serial.println("Servo motor is still detached");
```
Comment (per1234) [inline examples/MultipleServoMotors/MultipleServoMotors.ino]:
```suggestion
    Serial.println("Servo motor is was detached");
```

File: examples/MultipleServoMotors/MultipleServoMotors.ino [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  myservo.write(90);
+  delay(1000);  
+  myservo.write(180);
+  delay(1000);
+  } else{
+    Serial.println("Servo motor attachement failed");
+  }
+  
+  myservo.detach();        // detach the servo motor
+  if (myservo.attached()){ // cheack whether servo motor is attached or not
```
Comment (per1234) [inline examples/MultipleServoMotors/MultipleServoMotors.ino]:
```suggestion
  if (myservo.attached()){ // check whether servo motor is attached or not
```

File: examples/AnalogInput/AnalogInput.ino [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+void setup() 
+{ 
+  Serial.begin(9600);              // initializing serial communication with 9600 baud rate
+  myservo.attach(10);              // attaches the servo on pin 10 to the servo object
+  myservo.writeMicroseconds(500);  // setting initial angle to 0 (analog value = 500 [any value less than 700])
+  
+} 
+
+void loop() {
+  angle = myservo.read();          // read the value of the current angle on servo motor
```
Comment (per1234) [inline examples/AnalogInput/AnalogInput.ino]:
```suggestion
  byte angle = myservo.read();          // read the value of the current angle on servo motor
```
Compilation fails without variable declaration.

File: examples/MultipleServoMotors/MultipleServoMotors.ino [hunk 5]
```diff
@@ -0,0 +1,58 @@
+/*
+Controling multiple servo motors using attach() and detach()
+by Bhargav Patel <https://github.com/Engineer1999/>
```
Comment (per1234) [inline examples/MultipleServoMotors/MultipleServoMotors.ino]: It would be nice to have a description of the required circuit here (servos on pin 10 and 12).

**Review by per1234 — CHANGES_REQUESTED** [review]
Please always thoroughly test your code before submitting a pull request.

Please do a **Tools > Auto Format** if using the Arduino IDE or **Ctrl + B** if using Arduino Web Editor to ensure your formatting is consistent with that of the official Arduino example sketches.

Please remove pointless blank lines. Using blank lines to separate code into logical sections is fine, but random blank lines just make for more scrolling to read the code.

You use the term "analog" in the name of the "AnalogInput" sketch and in the comments, but there is nothing analog in this sketch. The servo is controlled by the duty cycle of a PWM signal. But the user doesn't need to know that. They only need to know that the position of the servo horn depends on the value passed to `writeMicroseconds()`.

**Comment by Engineer1999** [comment]
Thank you for your valuable feedback. I will make sure to follow your suggestions from the next pull requests. 
Also, @per1234 should I make a new pull request with modification as per your suggestions?

**Comment by per1234** [comment]
No, please don't make a new pull request. Just keep working on this one. Any commits you make to the added-library-examples-of-servo branch of your fork will automatically be added to this pull request.

**Comment by Engineer1999** [comment]
Sure. Thank you very much.

**Comment by Engineer1999** [comment]
@per1234 I have made a commit in the added-library-examples-of-servo branch with suggested changes.
Thank you for your guidance.
